pandas/_testing.py

```diff
@@ -917,9 +924,10 @@ def raise_assert_detail(obj, message, left, right, diff=None):
     elif is_categorical_dtype(right):
         right = repr(right)
 
-    msg = f"""{obj} are different
+    if isinstance(index, np.ndarray):
```

Comment:
what does this do?

Reply:
Oops - nothing.
Left over after refactor and should have been removed.  Will remove.  Thanks.